concepts/sepsis3/suspicion-of-infection.sql

```diff
@@ -0,0 +1,27 @@
+DROP TABLE IF EXISTS `physionet-data.mimic_derived.suspinfect_poe`;
+CREATE TABLE `physionet-data.mimic_derived.suspinfect_poe` as
+with abx as
```

Comment:
i would copy all the code from the abx_micro_poe table and put it here, and just have the one table ultimately named "suspicion_of_infection"

Reply:
I tried replacing the line `from physionet-data.mimic_derived.abx_micro_poe` with all the code from the table itself, e.g. `from ( <SQL from abx_micro_poe> )` but got some weird results. 

When I try to run the query in BQ _without_ the `CREATE TABLE` statement, the results are the same as when I use `from physionet-data.mimic_derived.abx_micro_poe`. 

However, when I run the query in BQ _with_ the `CREATE TABLE IF NOT EXISTS physionet-data.mimic_derived.suspinfect_poe` statement using `from ( <SQL from abx_micro_poe> )`, the `antibiotic_name` column no longer matches with the original. Moreover, each time I rerun the query to (re)create the `suspinfect_poe`, the `antibiotic_name` column changes. Would you happen to know why? Does BQ do some internal scheduling that doesn't necessarily guarantee the same output each time? I find it strange that only the `antibiotic_name` column changes. From a brief manual check, it seems that all other columns are consistent across runs of the same query.